benchflow-dsl/src/test/scala/cloud/benchflow/dsl/definition/types/time/TimeTest.scala

```diff
@@ -0,0 +1,42 @@
+package cloud.benchflow.dsl.definition.types.time
+
+import org.junit.{Assert, Test}
+import org.scalatest.junit.AssertionsForJUnit
+import net.jcazevedo.moultingyaml._
+import cloud.benchflow.dsl.definition.types.time.TimeYamlProtocol._
+
+import scala.util.Try
+
+/**
+  * @author Jesper Findahl (jesper.findahl@usi.ch) 
+  *         created on 14.03.17.
+  */
+class TimeTest extends AssertionsForJUnit {
```

Comment:
Updated in commit https://github.com/benchflow/benchflow/pull/67/commits/d1b3f4c4ca1357fbcc2031a3b776a86ca1ddf240